runelite-client/src/main/java/net/runelite/client/plugins/maxhit/config/CustomFormulaConfig.java

```diff
@@ -0,0 +1,233 @@
+	),
+
+	ORANGE_SALAMANDER(
+		MaxHitCalculator.CombatMethod.MAGIC,
+		new ArrayList<>(Collections.singletonList(
+			new EquipmentItemRequirement(new EquipmentSlotItem(EquipmentInventorySlot.WEAPON, new ArrayList<>(Collections.singletonList(
+				ItemID.ORANGE_SALAMANDER
+			))))
+		)),
+		(client, calculator) ->
+		{
+			int magicLevel = client.getBoostedSkillLevel(Skill.MAGIC);
+			return Math.floor(0.5 + magicLevel * (64.0 + 59.0) / 640.0);
+		}
+	),
+
+	RED_SALAMANDER(
+		MaxHitCalculator.CombatMethod.MAGIC,
+		new ArrayList<>(Collections.singletonList(
+			new EquipmentItemRequirement(new EquipmentSlotItem(EquipmentInventorySlot.WEAPON, new ArrayList<>(Collections.singletonList(
+				ItemID.RED_SALAMANDER
+			))))
+		)),
+		(client, calculator) ->
+		{
+			int magicLevel = client.getBoostedSkillLevel(Skill.MAGIC);
+			return Math.floor(0.5 + magicLevel * (64.0 + 77.0) / 640.0);
+		}
+	),
+
+	BLACK_SALAMANDER(
+		MaxHitCalculator.CombatMethod.MAGIC,
+		new ArrayList<>(Collections.singletonList(
+			new EquipmentItemRequirement(new EquipmentSlotItem(EquipmentInventorySlot.WEAPON, new ArrayList<>(Collections.singletonList(
+				ItemID.BLACK_SALAMANDER
+			))))
+		)),
+		(client, calculator) ->
+		{
+			int magicLevel = client.getBoostedSkillLevel(Skill.MAGIC);
+			return Math.floor(0.5 + magicLevel * (64.0 + 92.0) / 640.0);
+		}
+	),
+
+	DHAROK(
+		MaxHitCalculator.CombatMethod.MELEE,
+		new ArrayList<>(Collections.singletonList(new EquipmentItemSetRequirement(new EquipmentItemset(Arrays.asList(
+			new EquipmentSlotItem(EquipmentInventorySlot.HEAD, new ArrayList<>(Arrays.asList(
+				ItemID.DHAROKS_HELM,
+				ItemID.DHAROKS_HELM_100,
+				ItemID.DHAROKS_HELM_75,
+				ItemID.DHAROKS_HELM_50,
+				ItemID.DHAROKS_HELM_25,
+				ItemID.DHAROKS_HELM_0
+			))),
+			new EquipmentSlotItem(EquipmentInventorySlot.BODY, new ArrayList<>(Arrays.asList(
+				ItemID.DHAROKS_PLATEBODY,
+				ItemID.DHAROKS_PLATEBODY_100,
+				ItemID.DHAROKS_PLATEBODY_75,
+				ItemID.DHAROKS_PLATEBODY_50,
+				ItemID.DHAROKS_PLATEBODY_25,
+				ItemID.DHAROKS_PLATEBODY_0
+			))),
+			new EquipmentSlotItem(EquipmentInventorySlot.LEGS, new ArrayList<>(Arrays.asList(
+				ItemID.DHAROKS_PLATELEGS,
+				ItemID.DHAROKS_PLATELEGS_100,
+				ItemID.DHAROKS_PLATELEGS_75,
+				ItemID.DHAROKS_PLATELEGS_50,
+				ItemID.DHAROKS_PLATELEGS_25,
+				ItemID.DHAROKS_PLATELEGS_0
+			))),
+			new EquipmentSlotItem(EquipmentInventorySlot.WEAPON, new ArrayList<>(Arrays.asList(
+				ItemID.DHAROKS_GREATAXE,
+				ItemID.DHAROKS_GREATAXE_100,
+				ItemID.DHAROKS_GREATAXE_75,
+				ItemID.DHAROKS_GREATAXE_50,
+				ItemID.DHAROKS_GREATAXE_25,
+				ItemID.DHAROKS_GREATAXE_0
+			)))
+		))))),
+		(client, calculator) ->
+		{
+			int currentHP = client.getBoostedSkillLevel(Skill.HITPOINTS);
+			int maxHP = client.getRealSkillLevel(Skill.HITPOINTS);
+			int lostHP = maxHP - currentHP;
+
+			double initialMaxHit = calculator.calculateDefault();
+
+			double multiplier = (1.0 + lostHP / 100.0 * maxHP / 100.0);
```

Comment:
why cant this just be `1.0 + lostHP/100`

Reply:
The maximum HP seems to be important in the formula. For example: when your maxHP is 70 and you've lost 19, the multiplier will be 1.133. When your maxHP is 20 and you've lost 19 the multiplier will be 1.038. 

The only thing I'm uncertain about is whether there should be parenthesis or not. 
This is the current order: `(1.0 + ((lostHP / 100.0) * (maxHP / 100.0)))`